themes/gruvbox-dark.json

```diff
@@ -0,0 +1,937 @@
+      "scope":"keyword.operator",
+      "settings":{
+        "foreground":"#8ec07c"
+      }
+    },
+    {
+      "scope":"keyword.operator.new",
+      "settings":{
+        "foreground":"#fe8019"
+      }
+    },
+    {
+      "scope":"keyword.other.unit",
+      "settings":{
+        "foreground":"#b8bb26"
+      }
+    },
+    {
+      "scope":"metatag.php",
+      "settings":{
+        "foreground":"#fe8019"
+      }
+    },
+    {
+      "scope":"support.function.git-rebase",
+      "settings":{
+        "foreground":"#689d6a"
+      }
+    },
+    {
+      "scope":"constant.sha.git-rebase",
+      "settings":{
+        "foreground":"#b8bb26"
+      }
+    },
+    {
+      "name":"Types declaration and references",
+      "scope":[
+        "meta.type.name",
+        "meta.return.type",
+        "meta.return-type",
+        "meta.cast",
+        "meta.type.annotation",
+        "support.type",
+        "storage.type.cs",
+        "variable.class"
+      ],
+      "settings":{
+        "foreground":"#fabd2f"
+      }
+    },
+    {
+      "scope": "variable.this",
+      "settings": {
+        "foreground": "#d3869b"
+      }
+    },
+    {
+      "scope": [
+        "entity.name",
+        "entity.static",
+        "entity.name.class.static.function",
+        "entity.name.function",
+        "entity.name.class",
+        "entity.name.type"
+      ],
+      "settings": {
+        "foreground": "#fabd2f"
+      }
+    },
+    {
+      "name":"Function declarations",
+      "scope":[
+        "storage.type.function",
+        "entity.function",
+        "entity.name.function.static"
+      ],
+      "settings":{
+        "foreground":"#8ec07c"
+      }
+    },
+    {
+      "scope": "entity.name.function.function-call",
+      "settings": {
+        "foreground": "#8ec07c"
+      }
+    },
+    {
+      "scope": "support.function.builtin",
+      "settings": {
+        "foreground": "#fe8019"
+      }
+    },
+    {
+      "scope": [
+        "entity.name.method",
+        "entity.name.method.function-call",
+        "entity.name.static.function-call"
+      ],
+      "settings": {
+        "foreground": "#689d6a"
+      }
+    },
+    {
+      "scope": "brace",
+      "settings": {
+        "foreground": "#d5c4a1"
+      }
+    },
+    {
+      "name":"variables",
+      "scope":[
+        "meta.parameter.type.variable",
+        "variable.parameter",
+        "variable.name",
+        "variable.other",
+        "variable",
+        "string.constant.other.placeholder"
+      ],
+      "settings":{
+        "foreground":"#83a598"
+      }
+    },
+    {
+      "scope": "prototype",
+      "settings": {
+        "foreground": "#d3869b"
+      }
+    },
+    {
+      "scope": "storage.type.class",
+      "settings": {
+        "foreground": "#fb4934"
+      }
+    },
+    {
+      "scope": [
+        "entity.name.tag",
+        "punctuation.tag"
+      ],
+      "settings":{
+        "foreground":"#8ec07c",
+        "fontStyle":"bold"
```

Comment:
I'm a fan of using brighter colors for `bold` font instead of actually making the text look bold. I can create a variant of this one without the bold font though, so don't worry about it if you prefer using bold font, I have no problem with leaving this here.

However, with this scope, it's affecting CSS files as well, e.g
```css
.className div .anotherClassName {
  ...
}
```

in the above, `.className` and `.anotherClassName` aren't bold whilst `div` is, and I think not many people prefer it like that.

So, perhaps making this one specific to HTML & XML only?

Reply:
I didn't check css, i'll fix this also